sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -73,6 +73,21 @@ trait TimestampFormatterHelper extends TimeZoneAwareExpression {
   }
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_() - Returns the current timezone.",
+  examples = """
+    Examples:
+      > SELECT _FUNC_();
+       Asia/Shanghai
```

Comment:
You need to add this func in the ignored set: https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/expressions/ExpressionInfoSuite.scala#L148

Reply:
We don't need to print `an offset from UTC (e.g., +08:35)`? It seems presto/sql server do so (IMO printing it looks meaningful).

Reply:
Presto seems return the zone id, not see the offset. (checked both use and code).

Reply:
@ulysses-you . Oh, interesting. Does it mean Presto documentation has a bug?
- https://prestodb.io/docs/current/functions/datetime.html

> Returns the current time zone in the format defined by IANA (e.g., America/Los_Angeles) or as fixed offset from UTC (e.g., +08:35)

Reply:
I also read it and feel a little confused. Maybe it depend on the location since the doc said `zone or offset`.